cms/src/components/content/ContentOverview.vue

```diff
@@ -131,6 +171,104 @@ router.currentRoute.value.meta.title = `${capitaliseFirstLetter(titleType)} over
             :buttonPermission="canCreateNew"
             data-test="no-content"
         /> -->
+        <div class="flex h-14 w-full gap-2 rounded-t-md bg-white p-2 shadow-lg">
+            <input
+                type="text"
+                class="h-full w-2/4 rounded-md border-[1px] border-zinc-200 p-4 text-sm focus:border-zinc-950 focus:bg-zinc-50 focus:outline-none focus:ring-0"
+                name="search"
+                placeholder="Search..."
+                data-test="search-input"
+                v-model="searchTerm"
+            />
+            <div>
+                <div class="relative flex h-full gap-1">
+                    <button
+                        class="flex h-full w-10 flex-row content-center justify-center rounded-md border-[1px] focus:border-black focus:outline-none"
+                        @click="() => (showSortOptions = true)"
```

Comment:
Nitpick: No need to make this a method.

```suggestion
                        @click="showSortOptions = true"
```

would work too.